blog/_posts/2021-10-21-scala-3.1.0-released.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.0 released!
+---
+Hello from the Scala 3 team! It has already been six weeks since we have announced the release candidate for the first minor version after the initial release of Scala 3. Now, after two more RCs, we can confidently promote Scala 3.1.0-RC3 to a new stable release of the language. In other words, Scala 3.1.0 is officially out!
+
+## Compatibility notice
+
+This is a first *minor* release after the initial release of Scala 3.0. This has following consequences:
```

Comment:
perhaps insert somewhere in this section: "Scala 2 version numbers were epoch.major.minor, Scala 3 version numbers are major.minor.patch."

In my experience, it's remarkable how many people either don't know this yet, or maybe sort of vaguely remember hearing something of the sort at some point in the past, but could use a reminder.